src/pages/guide/integration/attributes/data-fixture.md

```diff
@@ -77,6 +81,76 @@ class QuoteTest extends \PHPUnit\Framework\TestCase
 }
 ```
 
+### Specifying the number of instances of the same configuration the data fixture should generate 
+
+<InlineAlert variant="info" slots="text" />
+
+At this time, the parameter is only available for Adobe Commerce developer, and will become publicly available with Adobe Commerce 2.4.6 release.
+
+Sometimes we need to generate several instances of a data fixture with exactly the same configuration.
+For such cases, we can use the `count` parameter, and set its value to the desired number of instances.
+
+```php?start_inline=1
+class ProductsList extends \PHPUnit\Framework\TestCase
+{
+   #[
+      DataFixture(ProductFixture::class, count: 3),
+   ]
+   public function testGetProductsCount(): void
+   {
+      $fixtures = DataFixtureStorageManager::getStorage();
+      $product1 = $fixtures->get('product1');
+      $product2 = $fixtures->get('product2');
+      $product3 = $fixtures->get('product3');
```

Comment:
no alias will be assigned to the fixtures if the alias is not explicitly defined